flake.nix

```diff
@@ -58,18 +63,35 @@
         };
       };
 
-      devShells.x86_64-linux.default = (project pkgsLocal).shellFor {
-        inherit (checks.x86_64-linux.pre-commit-check) shellHook;
-        withHoogle = false;
-        buildInputs =
-          with pkgsLocal;
-          [
-            haskellPackages.cabal-fmt
-            nixpkgs-fmt
-            nodejs
-          ]
-          ++ checks.x86_64-linux.pre-commit-check.enabledPackages;
-      };
+      devShells = forAllSystems (system:
+        let
+          pkgs = pkgsFor system;
+          proj = project pkgs;
```

Comment:
We generally avoid abbreviations in our naming. Suggest renaming the `project` function to `mkProject` so you can rename `proj` to `project`.